install/upgrade.php

```diff
@@ -113,6 +113,16 @@ public function _upgrade() {
 			}
 		}
 
+		/**
+		 * Fires once Polylang has been upgraded.
+		 *
+		 * @since 3.7
+		 *
+		 * @param $from_version Base version.
+		 * @param $to_version   New version.
+		 */
+		do_action( 'pll_upgrade', $this->options['version'], POLYLANG_VERSION );
```

Comment:
I believe that we don't need the additional parameters. They can be easily accessed by other ways if needed (as you did in Polylang Pro. What do you think?
```suggestion
		/**
		 * Fires after Polylang has been upgraded and before the new version is saved in options.
		 *
		 * @since 3.7
		 */
		do_action( 'pll_upgrade' );
```

Reply:
Sure!
I've proposed this parameters with third party developers in mind. But the lesser the better!